package-lock.json

```diff
@@ -2423,6 +2423,13 @@
         "base64-js": "^1.0.2",
```

Comment:
@MuhanguziDavid this file should be added to your git.ignore

Reply:
Thank you @mendozabree. `package-lock.json` has been added to `.gitignore`